lib/rules/media-query-no-invalid/index.mjs

```diff
@@ -52,16 +65,19 @@ const rule = (primary) => {
 
 			parseMediaQuery(atRule).forEach((mediaQuery) => {
 				if (isMediaQueryInvalid(mediaQuery)) {
-					// Queries that fail to parse are invalid.
-					complain(atRule, atRuleParamIndexValue, mediaQuery);
+					if (!shouldIgnoreNode(mediaQuery, secondaryOptions)) {
+						// Queries that fail to parse are invalid.
+						complain(atRule, atRuleParamIndexValue, mediaQuery);
+					}
 
 					return;
 				}
 
 				mediaQuery.walk(({ node, parent }) => {
-					// All general enclosed nodes are invalid.
 					if (isGeneralEnclosed(node)) {
-						complain(atRule, atRuleParamIndexValue, node);
+						if (!shouldIgnoreNode(mediaQuery, secondaryOptions)) {
+							complain(atRule, atRuleParamIndexValue, node);
+						}
```

Comment:
[suggestion] Minor. The same idea as <https://github.com/stylelint/stylelint/pull/8060/files#r1832706629>:

```suggestion
						if (shouldIgnoreNode(mediaQuery, secondaryOptions)) return;

						complain(atRule, atRuleParamIndexValue, node);
```